app/livechat/server/lib/Livechat.js

```diff
@@ -464,9 +492,7 @@ export const Livechat = {
 			rcSettings[setting._id] = setting.value;
 		});
 
-		settings.get('Livechat_history_monitor_type', (key, value) => {
-			rcSettings[key] = value;
-		});
+		rcSettings.Livechat_history_monitor_type = settings.get('Livechat_history_monitor_type');
```

Comment:
this can be added to array above ([line 490](https://github.com/RocketChat/Rocket.Chat/pull/22321/files#diff-a54a068ba97aea1ca55267c7e9150910a7f0ccaddd5e362968afb51e036317b8R490))

Reply:
actually, no it can't because it is not public =\